docs/guides/Guide-Intro.md

```diff
@@ -55,6 +55,8 @@ const SimpleApp = StackNavigator({
   Home: { screen: HomeScreen },
 });
 
+export default SimpleApp;
```

Comment:
Hi! Thanks for the PR! Unfortunately, this change is leaving out users who use the `react-native` CLI. This should ultimately be resolved with a PR that includes an 
 updated intro guide for CRNA and `react-native` CLI. See comments https://github.com/react-community/react-navigation/pull/2206 https://github.com/react-community/react-navigation/pull/1915#discussion_r123381209